canonicalwebteam/http/__init__.py

```diff
@@ -84,13 +96,15 @@ def request(self, method, url, **kwargs):
                 CONNECTION_FAILED_COUNTER.labels(domain=domain).inc()
 
             raise connection_error
+        except CircuitBreakerError as cbe:
```

Comment:
This feels a bit too trivial to be worth it. I think we should just let it raise a `CircuitBreakerError`.

Reply:
The idea of the custom error was just to avoid explicitly accessing pybreaker modules in the apps the consume the extension. If they want to do something with circuit breaker exceptions they could just import it from this module. Do you think it's not worth having this?

Reply:
Yeah I understand the theory, but it seems like overkill.

Reply:
I mean, you could consider `pybreaker.CircuitBreakerError` to be part of our module's interface which we can break or not deliberately. As in, we could literally document "This module will throw a `pybreaker.CircuitBreakerError`.

The benefit to that approach is that we are using an error type that actually exists in the community. Apps might well understand a `pybreaker.CircuitBreakerError` more natively than our own one. In a way, it seems a bit antisocial to create our own error type where an error already exists in the community.

Do you see what I mean? Or still disagree?

Reply:
Sounds sensible, let's just let it raise `pybreaker.CircuitBreakerError`.